extractor/filesystem/internal/cachedregexp/regex.go

```diff
@@ -0,0 +1,17 @@
+package cachedregexp
+
+import (
+	"regexp"
+	"sync"
+)
+
+var cache sync.Map
+
+func MustCompile(exp string) *regexp.Regexp {
```

Comment:
Internal linters require all public functions to have a comment starting with the function name, e.g. // MustCompile returns XYZ